.happy/terraform/modules/sfn_config/gisaid.wdl

```diff
@@ -4,44 +4,48 @@ workflow LoadGISAID {
     input {
         String docker_image_id = "aspen-gisaid"
         String aws_region = "us-west-2"
-        String db_data_bucket = "aspen-data-rdev"
-        String gisaid_ndjson_export_url = "https://www.epicov.org/epi3/3p/exp3/export/export.json.bz2"
-        String gisaid_ndjson_staging_bucket = "aspen-data-rdev"
-        String gisaid_ndjson_staging_key = "raw_gisaid_dump/test.zst"
+        String aspen_config_secret_name
+        String remote_dev_stack_name = ""
```

Comment:
looks great! I think we need to update our terraform to pass these values in to the SSM parameters as well though!

Reply:
I _think_ we only need to pass in aspen_config_secret_name.  Everything else is kind of optional.

Reply:
yeah, but we may as well set the stack name as well -- right now we're writing the inputs for a given env to an SSM parameter (#428 promotes these to cloudwatch event inputs) to make it easier to know how to invoke these SFN's: https://github.com/chanzuckerberg/aspen/blob/trunk/.happy/terraform/modules/sfn_config/main.tf#L8

They should be updated to add the additional params.